src/entrypoint/dealerEntrypoint.ts

```diff
@@ -0,0 +1,32 @@
+import { setupMongoConnection } from "../mongodb"
```

Comment:
you're in the entrypoint folder, so `dealer.ts` for the filename is probably good, no need to repeat entry point.